applications/app/views/fragments/galleryBody.scala.html

```diff
@@ -42,38 +44,29 @@ <h2 class="fc-container__header__title">
     @fragments.contentFooter(gallery, related, "media")
 </div>
 
-@galleryItem(adSlots: Seq[String], adInterval: Int, image: model.ImageAsset, rowNum: Int) = {
+@galleryItem(adSlots: Seq[String], adInterval: Int, image: model.ImageAsset, rowNum: Int, imageElement: model.ImageElement) = {
 
     <li id="img-@rowNum" class="gallery2__item js-gallery-item" data-link-name="Gallery item | @rowNum">
         <figure itemscope itemtype="http://schema.org/ImageObject">
 
-            @ImgSrc.imager(gallery.imageContainer(rowNum - 1), GalleryUpgradedImage).map { imgSrc =>
+            <a class="gallery2__img-container js-gallerythumbs @if(image.width >= image.height) {gallery2__img-container--landscape} else {gallery2__img-container--portrait}"
+                href="@LinkTo{@gallery.url#img-@rowNum}"
 
-                @GalleryInitialImage.bestFor(gallery.imageContainer(rowNum - 1)).map { url =>
+                data-link-name="Launch Gallery Lightbox" data-is-ajax>
 
-                    <a class="gallery2__img-container js-gallerythumbs js-delayed-image-upgrade @if(image.width >= image.height) {gallery2__img-container--landscape} else {gallery2__img-container--portrait}"
-                        href="@LinkTo{@gallery.url#img-@rowNum}"
-                        data-src="@imgSrc"
-                        data-link-name="Launch Gallery Lightbox" data-is-ajax>
+                <img class="gallery2__img js-gallery-img @if(image.width >= image.height) {gallery2__img--landscape} else {gallery2__img--portrait}"
+                    srcset="@ImgSrc.srcset(imageElement, GalleryMedia.Inline)"
+                    sizes="@GalleryMedia.Inline.sizes"
+                    src="@ImgSrc.getFallbackUrl(imageElement)"
```

Comment:
that's a good point, I just chose a fallback to retain the imager style behaviour, but it isn't always needed. I'll leave it for this one, but the best solution would be to provide a src for the first one or two gallery items. Teensy bit more logic.